src/main/scala/com/databricks/spark/csv/DefaultSource.scala

```diff
@@ -142,6 +142,14 @@ class DefaultSource
 
     val codec = parameters.getOrElse("codec", null)
 
+    val maxCharsPerColStr = parameters.getOrElse("maxCharsPerCol", "100000")
+    val maxCharsPerCol = try {
+      maxCharsPerColStr.toInt
+    } catch {
+      case e: Exception => throw new Exception("maxCharsPerCol must be a valid integer")
+    }
+
+
```

Comment:
Maybe single white space here if you are going to add some more commits? (See [scala-style-guide#blank-lines-vertical-whitespace](https://github.com/databricks/scala-style-guide#blank-lines-vertical-whitespace))